generators/spa/templates/Vue3JS/common/Constants/ModuleConstants.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace <%= fullNamespace %>.Constants
+{
+    /// <summary>
+    /// Constants used within the <%= friendlyName %> module.
+    /// </summary>
+    public class ModuleConstants
```

Comment:
@alejoroman0605 You ignored this comment.  CC: @ismaelTartola

Reply:
@WillStrohl I uploaded this class because I use constants in the code.
example:
```
  [HttpGet]
        [ActionName("GetResx")]
        public HttpResponseMessage GetResx(string filename)
        {
            var resx = new JObject();

            var resxRoot = $"{ModuleConstants.ResxPartialRoot}{filename}.resx";
            var filepath = HttpContext.Current.Server.MapPath(resxRoot);
            var resxDoc = new XmlDocument();
            resxDoc.Load(filepath);

            foreach (XmlNode dataNode in resxDoc.DocumentElement.SelectNodes(ModuleConstants.RootData))
            {
                var key = dataNode.Attributes[ModuleConstants.DataNodeAttributesName].Value;
                var val = Localization.GetString(key.ToString(), resxRoot);

                if (key.EndsWith(ModuleConstants.PointText, StringComparison.InvariantCultureIgnoreCase)) key = key.Substring(0, key.Length - 5);
                key = key.Replace(".", "_");

                resx.Add(key, val);
            }

            return Request.CreateResponse(resx);
        }
```

Reply:
@WillStrohl I use that class in the code for default values such as:
 ```
public HttpResponseMessage GetResx(string filename)
        {
            var resx = new JObject();

            var resxRoot = $"{ModuleConstants.ResxPartialRoot}{filename}.resx";
            var filepath = HttpContext.Current.Server.MapPath(resxRoot);
            var resxDoc = new XmlDocument();
            resxDoc.Load(filepath);

            foreach (XmlNode dataNode in resxDoc.DocumentElement.SelectNodes(ModuleConstants.RootData))
```

Reply:
Okay. As long as it's used. 👍🏽